template.tpl

```diff
@@ -60,59 +60,125 @@ ___TEMPLATE_PARAMETERS___
       }
     ]
   },
   {
     "type": "TEXT",
-    "name": "productIds",
-    "displayName": "Product IDs",
+    "name": "currency",
+    "displayName": "Currency",
     "simpleValueType": true,
-    "help": "Comma-separated list of product IDs that were part of the Order",
-    "valueHint": "IDs of products in order, e.g. 123,456,789",
+    "help": "Currency in which the Order Value was paid. \nList of supported currencies is available on \u003ca href\u003d\u0027https://fixer.io/symbols\u0027\u003ehttps://fixer.io/symbols\u003c/a\u003e",
+    "valueHint": "Currency of order, e.g. SEK, EUR, USD, etc",
     "valueValidators": [
       {
         "type": "NON_EMPTY",
-        "errorMessage": "You must provide Product IDs"
+        "errorMessage": "You must provide a Currency"
       }
     ]
+  },
+  {
+    "type": "GROUP",
+    "name": "setting",
+    "displayName": "Settings",
+    "subParams": [
+      {
+        "type": "CHECKBOX",
+        "name": "debug",
+        "checkboxText": "Enable debug mode",
+        "simpleValueType": true,
+        "help": "Logs debugging information on browser console"
+      },
+      {
+        "type": "TEXT",
+        "name": "scriptUrl",
+        "displayName": "Custom Tracking Library URL (Optional)",
+        "simpleValueType": true,
+        "canBeEmptyString": true,
+        "valueHint": "E.g. https://cdn.liveshopping.bambuser.com/metrics/bambuser.min.js",
+        "valueValidators": [
+          {
+            "type": "REGEX",
+            "args": [
+              "(^$|^(https?:\\/\\/)?([\\da-z\\.-]+\\.[a-z\\.]{2,6}|[\\d\\.]+)([\\/:?\u003d\u0026#]{1}[\\da-z\\.-]+)*[\\/\\?]?$)"
+            ],
+            "errorMessage": "Please insert a valid URL including!",
+            "enablingConditions": []
+          }
+        ],
+        "help": "Do not write anything here unless you have been given a custom tracking library URL!"
+      }
+    ],
+    "groupStyle": "ZIPPY_CLOSED"
   }
 ]
 
 
 ___SANDBOXED_JS_FOR_WEB_TEMPLATE___
 
 // Enter your template code here.
-const callInWindow = require('callInWindow');
+const injectScript = require('injectScript');
+const queryPermission = require('queryPermission');
 const copyFromWindow = require('copyFromWindow');
-const getCookieValues = require('getCookieValues');
-const log = require('logToConsole');
-
-var bbu = copyFromWindow('_bambuser');
-if (bbu != null && typeof(bbu) != 'undefined') {
-  var eventData = {};
-  eventData[bbu.params.EVENT.dataSelector] = 'purchase';
-  eventData[bbu.params.ORDER_ID.dataSelector] = data.orderId;
-  eventData[bbu.params.ORDER_VALUE.dataSelector] = data.orderValue;
-  eventData[bbu.params.CURRENCY.dataSelector] = data.currency;
-  eventData[bbu.params.ORDER_PRODUCTIDS.dataSelector] = data.productIds;
-  
-  bbu.collect(eventData);
-}
-else
+const logToConsole = require('logToConsole');
+
+// If the user chose to log debug output, initialize the logging method
+const log = data.debug
+  ? logToConsole
```

Comment:
If you want to make the log statements a bit cleaner you could log wrapper add the header. Something like:

```suggestion
  ? (...args) => logToConsole('🧪 Bambuser Debugger 🧪\n', ...args)
```